### PR TITLE
Purple theme

### PR DESCRIPTION
**Pull request by AAlakkad**
Looks nice with purple/pink background, e.g. http://alpha.wallhaven.cc/wallpaper/93486

### CODE REVIEW

File: themes/purple [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# vim: filetype=yaml
+---
+meta:
+  description: AUTOMATICALLY GENERATED THEME
```
Comment (acrisci) [inline themes/purple]: Please add a description here.

Comment (AAlakkad) [inline themes/purple]: Done, is it good?

**Comment by acrisci** [comment]
The colors look pretty good.

I think the focused inactive window color and the focused workspace button color should be different colors though. It's a little confusing on a multi-monitor setup to determine which workspace is focused based on the color. Maybe making the focused workspace button color background to be purple would help.

**Comment by AAlakkad** [comment]
Do you mean the light background with black text?

**Comment by acrisci** [comment]
I'm talking about `focused_workspace` under `bar_colors`.

I think the focused workspace should look like the focused titlebar in some way by using the same purple color.

**Comment by AAlakkad** [comment]
Can you see if it's good now? sorry I don't have dual monitor here, I can't test it.

**Comment by acrisci** [comment]
Yeah I like this a lot better :+1:

If you make any changes send more pull requests.

I'll release an update in probably about a week.

**Pull request merged**